### PR TITLE
Fix volumesFrom

### PR DESCRIPTION
**Pull request by KostyaSha**
Volumes must be an array.

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/StartContainerCmd.java [hunk 1]
```diff
@@ -36,6 +35,16 @@
 
 	public Capability[] getCapDrop();
 
+	public Boolean getReadonlyRootfs();
+
+	public int getCpuShares();
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/StartContainerCmd.java]: must be long...

**Comment by KostyaSha** [comment]
@marcuslinke if container api sync attempt is bad, then could you at least merge VolumesFrom fix?

**Comment by KostyaSha** [comment]
Sorry, lost integration tests. Will better split PRs.

**Comment by KostyaSha** [comment]
Split 2 commits to 2 PRs.

**Comment by KostyaSha** [comment]
re-triggering pr because it time-outed

**Comment by KostyaSha** [comment]
No idea what's wrong

**Comment by marcuslinke** [comment]
@KostyaSha Build sometimes times out with CircleCI. Don't know the root cause. I guess its related to connection pooling and timing somehow.

**Comment by marcuslinke** [comment]
Thanks!

**Pull request merged**